src/test/java/core/basesyntax/WriterToFileTest.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.WriterToFile;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.WriterToFileImpl;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriterToFileTest {
+    private static WriterToFile writer;
+    private static ReportService report;
+    private static String expected;
+    private static String validPathToFile;
+    private static String invalidPathToFile;
+
+    @BeforeClass
+    public static void beforeClass() {
+        writer = new WriterToFileImpl();
+        report = new ReportServiceImpl();
+        Storage.getDataBase().put(new Fruit("banana"), 10);
+        validPathToFile = "src/test/files/resultWriterService";
+        invalidPathToFile = "src/test/files1/resultWriterService";
```

Comment:
You can make these fields constants and define immediately or not use fields for paths at all